### PR TITLE
Adds new sha256sums for PyYAML 5.1 and new release

### PR DESCRIPTION
**Pull request by kushaldas**
Updates setup.py for new release of 0.1.3

It also adds the VERSION file as the right package data, and
fixes the following error due to the newline in that file.

/home/user/.local/lib/python3.5/site-packages/setuptools/dist.py:398: UserWarning: Normalizing '0.1.3
' to '0.1.3'
  normalized_version,

Fixes part of #26 

The sha256sum can be verified at https://github.com/freedomofpress/securedrop-debian-packaging/wiki/PyYAML-5.1

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ chardet==3.0.4 --hash=sha256:9f178988ca4c86e8a319b51aac1185b6fe5192328eb5a163c28
 furl==2.0.0 --hash=sha256:cc0eb8998dcc7c5b58bc8625891a9ff563e2765e112024fa3d1e3521481de8b6
 idna==2.7 --hash=sha256:954e65e127d0433a352981f43f291a438423d5b385ebf643c70fd740e0634111
 orderedmultidict==1.0 --hash=sha256:25489716d76d2cc8aa656bfb00cd40b6ca29d5e11ccde0db60c2b46ad52bb40a
-pyyaml==3.13 --hash=sha256:59fa6f097310e25248d5aa1d7b7e619ea22dfaada67e9c65262457b3e5b3a5c8
+pyyaml==5.1 --hash=sha256:b8d80623e9d4e348c59ea726ce3032a2eb15abca6a48d3828362d11c6014a0a7
```
Comment (redshiftzero) [inline requirements.txt]: re: your note about verification of the sha256sum, isn't this the sha256sum of the built wheel, i.e. it's not available in https://github.com/freedomofpress/securedrop-debian-packaging/wiki/PyYAML-5.1?

Comment (kushaldas) [reply requirements.txt]: Nope, the verification was done on the source, the final builtwheel sha256sum (in signed format) is in here https://github.com/freedomofpress/securedrop-debian-packaging/pull/35

Comment (kushaldas) [reply requirements.txt]: Sorry for making this PR first.

Comment (redshiftzero) [reply requirements.txt]: OK so yes this is the hash of the built wheel ;)

**Comment by kushaldas** [comment]
>lgtm! (also hmm we should add a changelog for this repo, will file followup for this)

yes, that is coming in the future PR :)

**Pull request merged**